target/linux/layerscape/base-files/etc/uci-defaults/05_fix-compat-version

```diff
@@ -10,6 +10,8 @@ case "$(board_name)" in
 	fsl,ls1021a-iot-sdboot | \
 	fsl,ls1021a-twr | \
 	fsl,ls1021a-twr-sdboot | \
+	fsl,ls1028a-rdb | \
```

Comment:
This will be not needed if you drop compat changes in image creation.

Reply:
Fixed in 2